src/status_im/contexts/communities/actions/channel_view_details/view.cljs

```diff
@@ -0,0 +1,113 @@
+        {:keys [ens-verified added?]} (rf/sub [:contacts/contact-by-address public-key])]
+    [contact-list-item/contact-list-item
+     {:on-press      #(rf/dispatch [:chat.ui/show-profile public-key])
+      :on-long-press show-profile-actions
+      :accessory     {:type     :options
+                      :on-press show-profile-actions}}
+     {:primary-name   primary-name
+      :secondary-name secondary-name
+      :public-key     public-key
+      :ens-verified   ens-verified
+      :added?         added?}
+     {:public-key public-key}]))
+
+(defn footer
+  []
+  [rn/view {:style style/footer}])
+
+(defn members
+  [items]
+  [rn/section-list
+   {:key-fn                            :public-key
+    :content-container-style           {:padding-bottom 20}
+    :get-item-layout                   (fn [_ index]
+                                         #js
+                                          {:length 200
+                                           :offset (* 200 index)
+                                           :index  index})
+    :content-inset-adjustment-behavior :never
+    :sections                          items
+    :sticky-section-headers-enabled    false
+    :render-section-header-fn          contact-list/contacts-section-header
+    :render-section-footer-fn          footer
+    :render-fn                         contact-item-render
+    :scroll-event-throttle             8}])
+
+(defn f-view
+  [{:keys [chat-id community-id theme]}]
+  (let [{:keys [description chat-name emoji muted
+                chat-type]
+         :as   chat}  (rf/sub [:chats/chat-by-id
+                               chat-id])
+        pins-count    (rf/sub [:chats/pin-messages-count chat-id])
+        items         (rf/sub [:communities/sorted-community-members-section-list
+                               community-id])
+        profile-color (rf/sub [:profile/customization-color])]
+    (fn [chat-id]
+      [rn/view {:style {:flex 1}}
+       [quo/page-nav
+        {:background :blur
+         :icon-name  :i/arrow-left
+         :on-press   #(rf/dispatch [:navigate-back])
+         :right-side [{:icon-name :i/options
+                       :on-press  #(rf/dispatch [:show-bottom-sheet
+                                                 {:content (fn [] [home.actions/chat-actions
+                                                                   chat
+                                                                   false
+                                                                   true])}])}]}]
+       [quo/text-combinations
+        {:container-style                 style/text-combinations
+         :title                           [quo/channel-name
+                                           {:channel-name chat-name
+                                            :unlocked?    true}]
+         :emoji                           (when (seq emoji) emoji)
```

Comment:
`emoji` is a string. Using `seq` on a string is not very precise nor the fastest because Clojure has to transform the string to an IndexedSeq. Use `(not (string/blank? emoji))`.

See results on the JVM, 40x faster. Longer strings give the same results.

```clojure
  (criterium/with-progress-reporting
    (criterium/bench
        (seq "Lorem Ipsum is simply")))
  ;; Evaluation count : 1442390700 in 60 samples of 24039845 calls.
  ;; Execution time mean : 38.201201 ns
  ;; Execution time std-deviation : 0.678546 ns
  ;; Execution time lower quantile : 36.352233 ns ( 2.5%)
  ;; Execution time upper quantile : 39.278308 ns (97.5%)
  ;; Overhead used : 5.455626 ns

  (criterium/with-progress-reporting
    (criterium/bench
        (not (string/blank? "Lorem Ipsum is simply"))))
  ;; Evaluation count : 10148449380 in 60 samples of 169140823 calls.
  ;; Execution time mean : 0.914537 ns
  ;; Execution time std-deviation : 0.071185 ns
  ;; Execution time lower quantile : 0.681369 ns ( 2.5%)
  ;; Execution time upper quantile : 0.984959 ns (97.5%)
  ;; Overhead used : 5.455626 ns
```

And by precise I mean the developer doesn't have any hint as to what type `emoji` is. I had to lookup the implementation of `text-combinations` to see that it's in fact a string.